docs/source/tutorials/qmd/07-error-correction.qmd

```diff
@@ -0,0 +1,623 @@
+ax1.plot([x.min(), x.max()], [x.min(), x.max()], 'k--', alpha=0.7, label='True')
+ax1.set_xlabel('True x')
+ax1.set_ylabel('Observed x (replicates)')
+ax1.set_title('True vs Observed Values')
+ax1.legend()
+ax1.grid(True, alpha=0.3)
+```
+## Implementing the Model in Liesel
+
+Now that we have our data we can define our distributions from above in Liesel,
+setting $\tau^2_{\mu} = 1000$ and $a_x = b_x = 0.001$.
+We will initialize $x$ using the mean of our observed measurements. We also need to assign a distribution to the observed
+measurements, where we choose a
+[`multivariate normal distribution`](https://www.tensorflow.org/probability/api_docs/python/tfp/distributions/MultivariateNormalFullCovariance?hl=en). Note that we are building
+a hierarchical model by providing a {class}`.Var` instance for the `loc` and `scale`
+arguments of the different distributions.
+
+```{python}
+#| label: distributions
+# Define hyperparameters for variance of x
+a_x = lsl.Var.new_param(0.001, name = "a_x")
+b_x = lsl.Var.new_param(0.001, name = "b_x")
+
+# Define prior for tau2_x using an Inverse Gamma distribution
+tau2_x_prior = lsl.Dist(tfd.InverseGamma, concentration = a_x, scale = b_x)
+tau2_x = lsl.Var.new_param(10.0, distribution = tau2_x_prior, name = "tau2_x")
+
+# Define hyperparameters for mu_x (mean of x)
+location_mu_x_prior = lsl.Var.new_param(0.0, name = "location_mu_x_prior")
+scale_mu_x_prior = lsl.Var.new_param(1000.0, name = "scale_mu_x_prior")
+
+# Define prior for mu_x using a Normal distribution
+mu_x_prior = lsl.Dist(tfd.Normal, loc = location_mu_x_prior, scale = scale_mu_x_prior)
+
+# Define mu_x as a parameter with the prior distribution
+mu_x = lsl.Var.new_param(0.0, distribution = mu_x_prior, name = "mu_x")
+
+# Define prior distribution for x
+x_prior_dist = lsl.Dist(tfd.Normal, loc = mu_x, scale = tau2_x)
+
+# Estimate x using the mean of replicates and assign a prior distribution
+x_estimated = lsl.Var.new_param(jnp.expand_dims(x_tilde.mean(axis = 1), -1), # initial estimation is the mean of the replicates
+                                distribution = x_prior_dist,
+                                name = "x_estimated")
+
+# Define likelihood model for measurement error
+measurement_dist = lsl.Dist(
+    tfd.MultivariateNormalFullCovariance,
+    loc = x_estimated,
+    covariance_matrix = sigma
+    )
+
+# Define x_tilde (observed replicates) as observed data in the probabilistic model
+x_tilde_var = lsl.Var.new_obs(
+    value = x_tilde,
+    distribution = measurement_dist,
+    name = "x_tilde"
+)
+```
+## The Distributional Regression Model
+
+Following that we will set up our distributional regression model using P-splines. Assuming a Gaussian with $N(\mu_i, \sigma_i^{2})$ for the response, the regression parameters are linked to the model parameters by $\mu_i = \eta^{\mu}(x_i)$ and $\sigma_i^{2} = exp(\eta^{\sigma^{2}}(x_i))$, where $exp(\cdot)$ ensures positivity of the scale parameter. The predictors are then given by:
```

Comment:
In math mode, write `\exp` instead of `exp`. It is an established function name. Usually, established function names are written upright, not in italics.